nifi-nar-bundles/nifi-py4j-bundle/nifi-py4j-bridge/src/main/java/org/apache/nifi/py4j/PythonProcess.java

```diff
@@ -267,6 +264,21 @@ private Process launchPythonProcess(final int listeningPort, final String authTo
         return processBuilder.start();
     }
 
+    String resolvePythonCommand() throws IOException {
+        final File pythonCmdFile = new File(processConfig.getPythonCommand());
+        final String pythonCmd = pythonCmdFile.getName();
+        File[] virtualEnvFileList = virtualEnvHome.listFiles((file, name) -> file.isDirectory() && (name.equals("bin") || name.equals("Scripts")));
```

Comment:
Recommend adding a comment regarding common names, and declaring the result as `final`:
```suggestion
        // Find command directories according to standard Python venv conventions
        final File[] virtualEnvCommandDirectories = virtualEnvHome.listFiles((file, name) -> file.isDirectory() && (name.equals("bin") || name.equals("Scripts")));
```